src/value.rs

```diff
@@ -118,3 +120,28 @@ impl From<Vec<u8>> for Value {
         Value::Base64(other)
     }
 }
+
+#[cfg(test)]
+mod tests {
+  use super::*;
```

Comment:
The rest of the code base is indended with 4 spaces instead of 2, can you fix this to also use 4 to be consistent?